src/c++/examples/simple_grpc_sequence_stream_infer_client.cc

```diff
@@ -74,18 +74,10 @@ Usage(char** argv, const std::string& msg = std::string())
 }
 
 void
-StreamSend(
+StreamSend_(
```

Comment:
It is not in our naming convention to end a function with '_'. What does that mean?
It is already inside and anonymous namespace so why not just call is "SendStream"?

Reply:
Its a different function. Should I call it SendStreamHelper?

Reply:
C++ overloading means you should be able to just call it SendStream